.github/pull_request_template.md

```diff
@@ -33,22 +34,8 @@
 
 - [ ] I have included a quick summary of the changes
 - [ ] I have indicated the related issue's id with `# <issue-id>` if changes are of type `fix`
-- [ ] I have included a high-level list of changes and their corresponding type
-    - Types: `feat` `fix` `docs` `style` `refactor` `perf` `test` `build` `ci` `chore` `revert`
-    - Breaking changes: `<type>!`
-    - Reference:
-      see [commit_msg_reference.md](https://github.com/norlab-ulaval/libpointmatcher/blob/develop/commit_msg_reference.md)
-      in the repository root for details
+- [ ] I have included a high-level list of changes and their corresponding types
+      (See [commit_msg_reference.md](https://github.com/norlab-ulaval/libnabo/tree/develop/doc/contributing/commit_msg_reference.md)
```

Comment:
The link is pointing to libnabo. Shouldn't it be libpointmatcher?

Reply:
@boxanm This is outdated. It was fixed in commit [6de0d75](https://github.com/norlab-ulaval/libpointmatcher/pull/550/commits/6de0d7585c6aa5e6283ed53de7482f1790281d52)